build.py

```diff
@@ -1,12 +1,34 @@
-def build_cython_extensions():
-    from Cython.Build import build_ext, cythonize
-    from setuptools.dist import Distribution
+import os
+import shutil
 
-    ext_modules = cythonize('dioptas/model/util/smooth_bruckner_cython.pyx')
-    dist = Distribution({"ext_modules": ext_modules})
-    cmd = build_ext(dist)
+from setuptools import Distribution
+from setuptools import Extension
+from setuptools.command.build_ext import build_ext
+
+
+extensions = [
+    Extension(
+        "dioptas.model.util.smooth_bruckner_cython",
+        ["dioptas/model/util/smooth_bruckner_cython.pyx"],
```

Comment:
`setuptools` detects and handles `pyx` files (see https://setuptools.pypa.io/en/latest/userguide/ext_modules.html#distributing-extensions-compiled-with-cython)